en/contributing/plugins.md

```diff
@@ -151,15 +151,15 @@ plugin **CMakeLists.txt** file (you can add multiple files).
 
 The example plugin adds the **example_impl_test.cpp** unit test as shown below:
 
-```cmake
+```cmake 

```

Comment:
@julianoes What do you have to update in CMakeLists.txt or other files to add unit tests for a)core b) core plugins c)external plugins (the text here is no longer correct). 

For core plugins it looks like maybe you add to this one: https://github.com/dronecore/DroneCore/blob/develop/cmake/unit_tests.cmake#L23

Reply:
Same way for external plugins? I only ask because if so, this is missing our unit test: https://github.com/dronecore/DroneCore/blob/develop/external_example/plugins/example/CMakeLists.txt

Reply:
Hi @julianoes 

OK, so I updated the docs [here](https://github.com/dronecore/docs/pull/95/commits/08eda8b953555c7e97eabb63666420f39a3f1081). Can you please check them. 
My concerns:
- This uses external example, so I am assuming that external example unit tests are added in the same way as normal plugins?
- The cmake scripts actually link unit_test_runner against the mission module [here](https://github.com/dronecore/DroneCore/blob/develop/cmake/unit_tests.cmake#L23). Doesn't the runner have to link against all modules it uses? I would have expected this to also be specified in the cmakelists of each plugin.